engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/organization.factory.js

```diff
@@ -14,7 +14,20 @@ angular.module('Bastion.organizations').factory('Organization',
         return BastionResource('/katello/api/v2/organizations/:id/:action',
             {id: '@id'},
             {
-                update: { method: 'PUT'},
+                get: {
+                    method: 'GET',
+                    transformResponse: function (data) {
+                        // Some fields cannot be PUT back
+                        var org;
+
+                        org = angular.fromJson(data);
+                        delete org.users;
+                        delete org.parent_id;
+                        delete org.parent_name;
+                        return org;
```

Comment:
I originally was just going to modify the org update from the manifests page to just send redhat_repository_url. However, any bastion page that updates an org would end up sending the GET values and would also break (as I discovered in one of my local plugins).

To be honest, I first tried to fix this in foreman itself (not allow 'users') but that was nack'ed. The parent params should also not be present in katello.

Thus! This seemed like the safest thing all around.